.github/workflows/sync_issue_to_jira.yaml

```diff
@@ -0,0 +1,21 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+name: Sync issue to Jira
```

Comment:
Just double sync with @carlcsaposs-canonical to make sure that this cross-checks correctly. I'm not sure why the issue did not work, but it could just be that in order to work, this should be merged to main. So it will work once in main, but since then, we got to trust our judgement

Reply:
Cool, thanks much!